fly_away/src/models/IFlight.d.ts

```diff
@@ -23,3 +23,19 @@ export interface IFLightAPIResponse {
     time: string;
     states: (string | number | boolean | number[])[][];
 }
+
+export interface IISSAPIResponse {
```

Comment:
some of these values may be null. Never trust an API.

Reply:
I will do a check at the API observable thanks.

Reply:
There are null checks where the data is used rather in the observable. Would you like me to change this?